contracts/fast/FastTopFacet.sol

```diff
@@ -52,5 +54,6 @@ contract FastTopFacet is AFastFacet {
     amount = LibHelpers.upTo(recipient, amount);
     // Transfer some eth!
     if (amount != 0) { recipient.transfer(amount); }
```

Comment:
Wouldn't this call back into `transfer` which in turn would fire a `DetailsChanged` type event?